src/documentation/markdown/2017/lar-data-fields.md

```diff
@@ -0,0 +1,378 @@
+## Public HMDA data fields with values and definitions
+
+### [activity\_year](#activity_year)
+
+- **Description:** The calendar year the data submission covers
+- **Values:**
+  - 2017
+
+### [respondent\_id](#respondent_id)
+
+- **Description:** One component of the primary key identifier for HMDA reporters in 2017 and prior. Respondent ID must be concatenated with Agency Code to guarantee uniqueness in the HMDA data. See Table 1 in the 2017 FIG for derivation instructions.
+- **Values:**
+  - Varying values
+
+### [agency\_code](#agency_code)
+
+- **Description:** A financial institution's associated Agency Code
+- **Values:**
+  - 1 - Office of the Comptroller of the Currency (OCC)
+  - 2 - Federal Reserve System (FRB)
+  - 3 - Federal Deposit Insurance Corporation (FDIC)
+  - 4 - Office of Thrift Supervision (OTS)
+  - 5 - National Credit Union Administration (NCUA)
+  - 7 - Department of Housing and Urban Development (HUD)
+  - 9 - Consumer Financial Protection Bureau (CFPB)
+
+### [loan\_type](#loan_type)
+
+- **Description:** The type of covered loan or application
+- **Values:**
+  - 1 - Conventional (not insured or guaranteed by FHA, VA, RHS, or FSA)
+  - 2 - Federal Housing Administration insured (FHA)
+  - 3 - Veterans Affairs guaranteed (VA)
+  - 4 - USDA Rural Housing Service or Farm Service Agency guaranteed (RHS or FSA)
+
+### [property\_type](#property_type)
+
+- **Description:** Classification of dwelling
+- **Values:** 
+ - 1 - Single family with 1-4 units
+ - 2 - Manufactured housing
+ - 3 - Multifamily housing
+
+### [loan\_purpose](#loan_purpose)
+
+- **Description:** The purpose of covered loan or application
+- **Values:**
+  - 1 - Home purchase (one-to-four family)
+  - 2 - Home improvement (one-to-four family)
+  - 3 - Refinancing (home purchase or home improvement,
+ one-to-four family) 
+  - 4 - Multifamily dwelling (home purchase, home improvement, and refinancings)
+
+### [occupancy_type](#occupancy_type)
+
+- **Description:** Occupancy type for the dwelling
+- **Values:**
+  - 1 - Principal residence
```

Comment:
for 2017 these enumerations are:
1.Owner-occupied as a principal dwelling
2.Not owner-occupied
3.Not applicable